### ISSUE TITLE
Bug with time spent on YouTube

### ISSUE THREAD

**Issue by mcurmei627** [Algorithmic-Personalization/ytdpnl-server#4]
![Image](https://github.com/Algorithmic-Personalization/ytdpnl-server/assets/19541391/ab36a860-1f21-4a00-996e-f2e3cd7778e4)

There seems to be much less time spent on YouTube compared to watch time, is that last column tracking time spent with an active tab.

For context, with this user I was actually watching a playlist of videos on autoplay.

**Comment by djfm** [comment]
No, it should track watch time whether the tab is active or not, I haven't investigated the impact of tab activity on watch time events, I guess they are fired whether the tab is active or not since videos keep playing.

Still unsure whether the `tab_active` property is reliable. My gut feeling is that there are gonna be inconsistencies between different combinations of:
- operating system
- desktop environment, window manager...
- desktop settings (like, "raise windows on hover" probably changes something)
- browser vendor, version
- number of physical screens
- number of browser windows...

`For context, with this user I was actually watching a playlist of videos on autoplay.`

I think this makes sense, because watch time will be tracked any time a video plays, even a short probably (to be checked), but if the user never clicks, they're actually gonna experience the native YouTube behavior (especially if watching playlists, since playlist related events are tracked but playlists are unaltered by the extension).

I think the real question here is that of autoplay and what we can / want to do in the various cases when it happens.
